src/index.html

```diff
@@ -17,6 +29,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+          class="image"
+        />
+      </a>
+
+      <nav clas="nav">
```

Comment:
There is a typo in the 'nav' element. The attribute 'clas' should be 'class'. This typo will prevent the CSS styles associated with the 'nav' class from being applied.